simln-lib/src/lib.rs

```diff
@@ -164,6 +164,8 @@ pub type Interval = ValueOrRange<u16>;
 /// This is constructed during activity validation and passed along to the [Simulation].
 #[derive(Debug, Clone)]
 pub struct ActivityDefinition {
+    /// Optional name/identifier for this activity
```

Comment:
nit: here and above, `name/identifier` isn't helpful - since we're requiring that these be unique I think we can just say identifier in our comments.

Should also terminate comments with `.` - applies in various places for this PR